src/schema/schema.graphql

```diff
@@ -0,0 +1,53 @@
+type User {
```

Comment:
@ronanfitz are you actually utilizing this `graphql` file? I'm still unclear how it actually gets used. I know that you are using the `createSchema` function passing in a query string, which seems like it would create the schema. Either let me know that this file is actually necessary, or, remove it please. Thanks!